backend/src/main/java/com/eyetrade/backend/model/entity/Comment.java

```diff
@@ -0,0 +1,41 @@
+package com.eyetrade.backend.model.entity;
+
+import lombok.Data;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+/**
+ * Created by Emir Gökdemir
+ * on 24 Kas 2019
+ */
+
+@Data
+@Entity
+@Table(name = "comment")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO,generator = "system-uuid")
+    @GenericGenerator(name = "system-uuid", strategy = "uuid2")
+    @Column(name = "id")
+    private UUID id;
+
+    private String title;
+
+    @Column(length = 1024)
+    private String content;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User userId;
```

Comment:
I think the naming "userId" is a little bit confusing. Maybe you can change it to user later. But not a big problem :)

Reply:
Done